### ISSUE TITLE
Confused how to run on MacOS

### ISSUE THREAD

**Issue by divan** [ILikeAI/AlwaysReddy#100]
Hi, fantastic project! I followed installation steps for MacOS, all went smoothly, but now stuck with two things:

1) Hotkeys

There is no 'default' hotkeys for MacOS in config.py comments section.

Defining hotkeys is often confusing topic in other apps (is it cmd or CMD, 'super'-key etc), but then here is a line in config.py that says:
```
# On Mac/Linux, a hotkey cannot overlap another (e.g. cmd+e and cmd+shift+e)
```

What does it mean? Should I check system hotkeys and make sure that chosen hotkey is not overlapping?

I added 'cmd-shift-m', but nothing happens. I don't know if something is wrong with hotkey or it's another issue. (That's why it would be cool to have default hotkeys)

2) I see a line in the output, which probably explains why hotkeys is not working.
```
This process is not trusted! Input event monitoring will not be possible until it is added to accessibility clients.
```

The concept of adding apps to the trusted list is not unusual on MacOS, but quick search on Readme reveals nothing about it. How do I do it? Should go to settings and add that bash script `run_AlwaysReddy.sh` to the list? Or it's something else completely?

Bottom line, it's a bit frustrating how smoothly the installation went, and how it's impossible to use the app out of the box because of the main UI promise (just hotkeys) needs some non-documented configuration first.

**Comment by divan** [comment]
Anyone using this project on MacOS could share their configuration and setup perhaps?

**Comment by divan** [comment]
Figured out second issue - need to add terminal to the `Accessibility` list in `Settings->Privacy` (in my case it's 'Warp' terminal)
![Screenshot 2024-11-28 at 13 44 10](https://github.com/user-attachments/assets/7864bd5c-df76-4e09-8acd-9f62411978cd)

**Comment by divan** [comment]
Next, we need to add terminal to the `Input Monitoring` too in `Settings`:
![Screenshot 2024-11-28 at 13 49 37](https://github.com/user-attachments/assets/5e7e2edf-1a80-4af2-ad2c-c1af1e7ea56d)

**Comment by ghNico** [comment]
@divan can you run AlwaysReddy with this setup? I wanted to try this out, but keep getting stuck after i told anything. 
![image](https://github.com/user-attachments/assets/f9ee2249-0c6e-400a-bd59-020aaf073f50)
My Configuration is local lm studio and it worked only one time but without my told-text 
![image](https://github.com/user-attachments/assets/f7c74561-1976-4f99-9f37-f65b7929a60c)
It seems like its getting stuck during the fasterWhisper run, when i try to install via setup.py the 2nd option TransformerWhisper it installs nothing. I think the project is a bit broken. Would love to hear back from you.

**Comment by scalar27** [comment]
I tried AlwaysReddy several months ago and ran into problems (I have a M1 Mac) but then found https://github.com/DissonanceTK/MacReddy which is a fork.  It works great, but has not been updated in a while.  So at this point I may be missing new features or functionality.  Regarding hotkeys, changed one line in config.py to RECORD_HOTKEY = 'alt+cmd' since that is easier to type.  Hope this helps a bit.
